pyensembl/species.py

```diff
@@ -169,11 +169,10 @@ def check_species_object(species_name_or_object):
 mouse = add_species(
     latin_name="mus_musculus",
     synonyms=["mouse", "house mouse"],
-    # TODO: fix release range
-    # The range of releases given for GRCm38  isn't right and we don't have
-    # ranges for other assemblies.
     reference_assemblies={
-        "GRCm38": (78, MAX_ENSEMBL_RELEASE)
```

Comment:
I just flipped through different release directories on the Ensembl FTP
server until I found the change-point from NCBIM37 to GRCm38 (which
happened after Ensembl 67).

On Tue, May 10, 2016 at 4:32 PM, Tavi Nathanson <notifications@github.com>
wrote:

> In pyensembl/species.py
> <https://github.com/hammerlab/pyensembl/pull/146#discussion_r62746619>:
>
> >      reference_assemblies={
> > -        "GRCm38": (78, MAX_ENSEMBL_RELEASE)
>
> Can you walk me through the numbers, here?
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly or view it on GitHub
> <https://github.com/hammerlab/pyensembl/pull/146/files/7176d074bad3915a5235cc5a1672d85ff9b649c0#r62746619>
>